src/patternfly/demos/Page/page-template-header-tools-elements.hbs

```diff
@@ -8,9 +8,8 @@
     {{/button}}
   {{/page-header-tools-group}}
   {{#> page-header-tools-group}}
-    {{#> button button--modifier="pf-m-plain pf-m-mobile" button--attribute='aria-label="Overflow actions"'}}
-      {{> button-icon button-icon--type="ellipsis-v"}}
-    {{/button}}
+    {{#> dropdown id="dropdown-kebab-right-aligned-1" dropdown--IsActionMenu="true" dropdown-menu--modifier="pf-m-align-right" dropdown-toggle--modifier="pf-m-plain" dropdown--HasKebabIcon="true" aria-label="Actions"}}
```

Comment:
you need to add the `.pf-m-mobile` class so it only shows as you shrink the page down. Currently it's shown on desktop.

<img width="434" alt="screen shot 2019-02-19 at 4 32 05 pm" src="https://user-images.githubusercontent.com/35148959/53052342-0c585400-3464-11e9-82be-e3718f4bc6d3.png">

Reply:
Yep fixed